src/services/Store/store/helpers.ts

```diff
@@ -94,30 +90,6 @@ export const mergeAssets = (inbound: ExtendedAsset[], original: ExtendedAsset[])
     })
     .concat(inbound.filter((i) => !original.find((o) => o.uuid === i.uuid)));
 
-/**
- * Compare json to import with our persist state
- */
-export const canImport = (toImport: Partial<LocalStorage>, store: LocalStorage) => {
-  if (toImport.version !== store.version) {
-    return false;
-  } else {
-    // Check that all the keys in the store exist in the file to import
-    const diff = difference(keys(store), keys(toImport));
-    return diff.length === 0;
-  }
-};
-
-export const migrateConfig = (toImport: Partial<LocalStorage>) => {
-  return {
-    ...toImport,
-    // @ts-expect-error rates are present in settings on data to be migrated, want to move it at root of persistence layer
-    rates: toImport.settings?.rates ? toImport.settings.rates : toImport.rates,
```

Comment:
We need to support these cases in our new migrations, as with the current logic the `rates` fx. will be nuked when importing a settings file which has already migrated its rates.

Reply:
IMO if the rates are nuked it's not that bad they will be re-fetched right after the import. This problem will be fixed by adding the persist version to the export file

Reply:
I would prefer if we support these small migrations for stability 😄